bob/src/cluster/quorum.rs

```diff
@@ -32,31 +32,31 @@ impl Quorum {
 
     async fn put_at_least(&self, key: BobKey, data: &BobData) -> Result<(), Error> {
         let mut local_put_ok = 0_usize;
-        let at_least = self.quorum;
+        let mut at_least = self.quorum;
         let mut failed_nodes = Vec::new();
-        let (vdisk_id, disk_path) = self.mapper.get_operation(key);
-        let (tasks, errors) =
-            if let Some(disk_path) = disk_path {
-                debug!("PUT[{}] ~~~PUT TO {} REMOTE NODES AND LOCAL NODE~~~", key, at_least - 1);
-                let (remote_put, local_put) = tokio::join!(
```

Comment:
Parallelization should be returned. This was an important optimization.
I think this can be done by adding a new function to `operations.rs` that puts in multiple paths. In this case, here the `tokio::join!`  with `put_remote_nodes` and this new function can be called